test/e2e/go.mod

```diff
@@ -161,7 +167,15 @@ require (
 	github.com/coreos/butane v0.18.0
 	github.com/docker/docker v20.10.24+incompatible
 	github.com/google/goexpect v0.0.0-20210430020637-ab937bf7fd6f
+	github.com/openshift-kni/k8sreporter v1.0.4
+	github.com/ovn-org/ovn-kubernetes/go-controller v0.0.0-20230914190234-9cbdd02db9c6
+	go.universe.tf/metallb v0.0.0-00010101000000-000000000000
 	google.golang.org/grpc v1.53.0
-	kubevirt.io/api v1.0.0-alpha.0
-	kubevirt.io/client-go v1.0.0-alpha.0
+	kubevirt.io/api v1.0.0
+	kubevirt.io/client-go v1.0.0
+)
+
+replace (
+	github.com/onsi/ginkgo/v2 => github.com/onsi/ginkgo/v2 v2.4.0
+	go.universe.tf/metallb => github.com/metallb/metallb v0.1.1-0.20230908122239-a5f74ed4c45c
 )
```

Comment:
Why do we need these replaced?

Reply:
yes @jcaamano , good spot on. we can remove ginkgo/v2 from replace directive (if you don't mind can we do it as part of next PR which removes ginkgo reporter deprecation), but for metallb, the module name is `go.universe.tf/metallb`  whereas github location is `github.com/metallb/metallb`. so it needed replace.

Reply:
ok, just removed replace directive for ginkgo/v2, hope it shouldn't be a problem.